src/main/webapp/app/exam/overview/summary/exercises/quiz-exam-summary/quiz-exam-summary.component.ts

```
@@ -127,8 +127,8 @@ export class QuizExamSummaryComponent implements OnChanges {
      * We only show the notice when there is a publishResultsDate that has already passed by now and the result is missing
      */
     get showMissingResultsNotice(): boolean {
-        if (this.exam && this.exam.publishResultsDate && this.quizParticipation.studentParticipations && this.quizParticipation.studentParticipations.length > 0) {
-            return dayjs(this.exam.publishResultsDate).isBefore(this.serverDateService.now()) && !this.result;
+        if (this.exam() && this.exam().publishResultsDate && this.quizParticipation().studentParticipations && this.quizParticipation().studentParticipations!.length > 0) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)